### DISCUSSION TITLE
I'm getting this error when I try to launch my mod, any idea what might be causing it?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#3029] [Mod Dev Support] [answered]
I'm getting this error when I try to launch my mod, any idea what might be causing it?
The error is quite vague
https://gist.github.com/TechPro424/160aac9dbb5373f0d04535c0973c8921

**Comment by zatrit** [comment]
Same issue

Upd: [crash-2023-05-12_19.04.54-client.txt](https://github.com/FabricMC/fabric/files/11465966/crash-2023-05-12_19.04.54-client.txt)

**Comment by zatrit** [comment]
I found a solution.

Try to exclude Fabric API from your dependencies like this:
```groovy
modApi("com.github.Fallen-Breath:conditional-mixin:${project.conditional_mixin_version}"){
        exclude group: "net.fabricmc"
        exclude group: "net.fabricmc.fabric-api"
}
```

**Comment by TechPro424** [comment, marked as answer]
@zatrit Thanks for your answer 👍 

In my case, however, the issue was that I forgot to change the main entrypoint of the mod from ExampleMod to AuctionHouse
(Solved it long back, forgot to post here, my bad)
